### ISSUE TITLE
`Spergel` `stepk` has two powers of the scale radius

### ISSUE THREAD

**Issue by beckermr** [GalSim-developers/GalSim#1324]
It appears that galsim has two factors of `r0` being applied when computing the `stepk` for the `Spergel` profile.

The first power is here:

https://github.com/GalSim-developers/GalSim/blob/120eab3b413aa7bc1447e72b8e3df4e5f4d6ac43/src/SBSpergel.cpp#L101-L102

but then the scale radius is applied again here:

https://github.com/GalSim-developers/GalSim/blob/120eab3b413aa7bc1447e72b8e3df4e5f4d6ac43/galsim/spergel.py#L206-L211

I compared to the implementation in jax-galsim, and I can only get them to agree if I use a second factor of `r0` within jax-galsim like this

```python
    @property
    def _stepk(self):
        R = calculateFluxRadius(1.0 - self.gsparams.folding_threshold, self.nu)
        R *= self._r0**2
        # Go to at least 5*hlr
        R = jnp.maximum(R, self.gsparams.stepk_minimum_hlr * self.half_light_radius)
        return jnp.pi / R
```

Note that the jax-galsim `calculateFluxRadius` function is in normalized units, and so should have only one factor of `r0`.

**Comment by rmjarvis** [comment]
I concur.  Looks like this error was introduced when trying to move more of the implementation up to python.  The pure C++ code looks right, but the python code has a spurious extra r0.  And we apparently didn't have a test that would catch the change.

**Comment by jmeyers314** [comment]
Yep.  Tests pass either way.  Maybe we should check `stepk /propto scale_radius`.

**Comment by rmjarvis** [comment]
propto 1/scale_radius.  But yeah, we should add a test about this.
